tasmota/xdrv_05_irremote_full.ino

```diff
@@ -470,6 +470,23 @@ uint32_t IrRemoteCmndIrSendRaw(void)
       return IE_INVALID_RAWDATA;
     }   // Parameters must be at least 3
 
+#ifdef IR_GC
+   if (strcmp(str, "gc") == 0 || strcmp(str, "GC") == 0) {  //if first parameter is gc then we process global cache data else it is raw
```

Comment:
You won't catch Gc or gC. Usse strcasecmp instead.

Reply:
True and done. I also removed the ifdef ir_gc check since really this section is only enabled when irremote_full is defined which enables most protocols anyhow so will not really impact resources in any large extent at this point.  Up to you.